src/notification-groups/risc-v.md

```diff
@@ -0,0 +1,22 @@
+# RISC-V notification group
+
+**Github Label:** [O-riscv]
+
+[O-riscv]: https://github.com/rust-lang/rust/labels/O-riscv
+
+This list will be used to ask for help both in diagnosing and testing
+RISC-V-related issues as well as suggestions on how to resolve
+interesting questions regarding our RISC-V support.
+
+The group also has an associated Zulip stream ([`#t-compiler/risc-v`])
+where people can go to pose questions and discuss RISC-V-specific
+topics.
+
+So, if you are interested in participating, please sign up for the
+RISC-V group! To do so, you open a PR against the [rust-lang/team]
```

Comment:
```suggestion
RISC-V group! To do so, open a PR against the [rust-lang/team]
```

Reply:
Thanks! Applied it to ARM and Windows instructions as well.